src/components/RelatiesKoppelingenTekstueel/RelatiesKoppelingenTekstueel.js

```diff
@@ -104,6 +113,17 @@ function RelatiesKoppelingenTekstueel({
     }
 }
 
+/**
+ * Component that renders the ListItem component, which displays a list of Verordening items which each item contain a title and description.
+ *
+ * @component
+ *
+ * @param {string} property - Parameter containing the property value.
+ * @param {UUID} UUID - Parameter containing the UUID value.
+ * @param {Array} connectionPropertiesColors - Parameter containing a collection of connection property colors.
```

Comment:
type object

Reply:
I've changed the parameter type to object.